frontend/src/utils/browserHistory.ts

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history';
+
+export default createBrowserHistory();
```

Comment:
Whats the purpose of this file? Is it just to rename the import in `index.tsx`?

Reply:
createBrowserHistory is a function, here you're calling it and returning the output - whatever it is
Doesn't look like renaming, more like initialising?

Reply:
Yeah this just initialises it, and a few resources online seem to do this by creating it in the utils folder. I'm going to rename it back to `history` as this also seems to be common practice.

This is also how Auth0 use it in their documentation when setting up the front-end.